content/get-started/using-git/dealing-with-special-characters-in-branch-and-tag-names.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Dealing with special characters in branch and tag names
+intro: 'Git is very permissive about what characters are allowed in branch and tag names. When using Git from a command-line shell, you may need to escape or quote special characters.'
+versions:
+  fpt: '*'
+  ghes: '*'
+  ghae: '*'
+shortTitle: Special characters in names
+---
+
+## About branch and tag names
+
+Most repositories use simple branch names, such as `main` or `update-icons`. Tag names also usually follow a basic format, such as a version number like `v1.2.3`. Both branch names and tag names may also use the path separator (`/`) for structure, for example `area/item` or `level-1/level-2/level-3`. Other than some exceptions &mdash; such as not starting or ending a name with a slash, or having consecutive slashes in the name &mdash; Git has very few restrictions on what characters may be used in branch and tag names. For more information, see "[git-check-ref-format](https://git-scm.com/docs/git-check-ref-format)" in the Git documentation.
+
+## Why escaping special characters is necessary
+
+When using a CLI, you might have situations where a branch or tag name contains special characters that have a special meaning for your shell environment. These characters must be quoted or escaped for you to use them safely in a Git command, otherwise running the command may have unintended effects.
```

Comment:
```suggestion
When using a CLI, you might have situations where a branch or tag name contains special characters that have a special meaning for your shell environment. To use these characters safely in a Git command, they must be quoted or escaped, otherwise the command may have unintended effects.
```